example/extending/newlang/5-hello-python/build.mill

```diff
@@ -0,0 +1,158 @@
+// == Basic Python Pipeline 
+
+// This Example demonstrates integration of https://www.python.org[Python]
+// compilation into a Mill build to compile https://www.python.org[Python] Scripts. Mill
```

Comment:
As a rule of thumb, you should not link the same keyword more than once per page

Reply:
i thought a lot for choosing name 😅

finally i choosed `Python Integration in Mill`